BUILDING.md

```diff
@@ -0,0 +1,68 @@
+## Build the development environment
+
+The first prerequisite of properly building SwarmKit targets is to have a Go
+development environment setup. Please follow [How to Write Go Code](https://golang.org/doc/code.html)
+for proper setup. If done correctly, you should have a GOROOT and GOPATH set in the
+environment.
+
+If a Go development environment is setup, one can use `go get` to install the
+`swarmkit`:
+
+    go get github.com/docker/swarmkit/...
+
+The above will install the source repository into the `GOPATH`. Any binaries
+should be built into `$GOPATH/bin`. Alternatively, use the above with `-d` to
+just download the source.
+
+> __NOTE:__ While you do not need to use `go get` to checkout the SwarmKit
```

Comment:
It is not mandatory to use `go get` to checkout the SwarmKit project. However, for these instructions to work, you need to check out the project to the correct subdirectory of the `GOPATH`: `$GOPATH/src/github.com/docker/swarmkit`.